src/manifest.ts

```diff
@@ -1052,13 +1064,16 @@ async function parseReleasedVersions(
  * Find the most recent matching release tag on the branch we're
  * configured for.
  *
- * @param {string} prefix - Limit the release to a specific component.
- * @param {boolean} preRelease - Whether or not to return pre-release
- *   versions. Defaults to false.
+ * @param github GitHub client instance.
+ * @param {string} targetBranch Name of the scanned branch.
+ * @param releaseFilter Validator function for release version. Used to filter-out SNAPSHOT releases for Java strategy.
+ * @param {string} prefix Limit the release to a specific component.
+ * @param pullRequestTitlePattern Configured PR title pattern.
  */
 async function latestReleaseVersion(
   github: GitHub,
   targetBranch: string,
+  releaseFilter: (version: Version) => boolean,
```

Comment:
This is technically a breaking change to the interface (`Manifest` is exported from the index and this is a public method). Can we set a default implementation?

Reply:
This should not be breaking, this is actually a private method used only directly in the `manifest.ts` itself. 
But possible breaking change is done to `Strategy`, where this method is added.  That is, if anyone would implement the interface, and not extend the `BaseStrategy`. I highly doubt that. Still, I will make the `Strategy.isValidRelease()` optional to avoid that breaking change.

Reply:
I've changed the interface `Strategy` in 7da1b08428eef15ff25fc281423e66a861cb3b3a - the `lastestReleaseVersion` signature is left intact however.